packages/tools/viewer/src/viewer.ts

```diff
@@ -280,6 +280,16 @@ export type ViewerOptions = Partial<{
      * Enabled by default.
      */
     autoSuspendRendering: boolean;
+
+    /**
+     * Automatically rotates a 3D model or scene without requiring user interaction.
+     */
+    cameraAutoOrbit?: Partial<CameraAutoOrbit>;
+
+    /**
+     * Boolean indicating if the scene must use right-handed coordinates system.
+     */
+    useRightHandedSystem?: boolean;

```

Comment:
These are implicitly optional because they are in a `Partial` type.
```suggestion

    /**
     * Automatically rotates a 3D model or scene without requiring user interaction.
     */
    cameraAutoOrbit: Partial<CameraAutoOrbit>;

    /**
     * Boolean indicating if the scene must use right-handed coordinates system.
     */
    useRightHandedSystem: boolean;

```